bundles/org.openhab.binding.bluetooth.govee/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="bluetooth"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
+	<thing-type id="goveeHygrometer">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="roaming"/>
+			<bridge-type-ref id="bluegiga"/>
+			<bridge-type-ref id="bluez"/>
+		</supported-bridge-type-refs>
+
+		<label>Govee Hygrometer</label>
+		<description>Govee Thermo-Hygrometer</description>
+
+		<channels>
+			<channel id="rssi" typeId="rssi"/>
+			<channel id="battery" typeId="system.battery-level"/>
+
+			<channel id="temperature" typeId="govee-temperature"/>
+			<channel id="humidity" typeId="system.atmospheric-humidity"/>
+		</channels>
+
+		<representation-property>address</representation-property>
+
+		<config-description>
+			<parameter-group name="calibration">
+				<label>Calibration</label>
+				<description>Sensor calibration settings.</description>
+				<advanced>true</advanced>
+			</parameter-group>
+
+			<parameter name="address" type="text">
+				<label>Address</label>
+				<description>Bluetooth address in XX:XX:XX:XX:XX:XX format</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" unit="s">
+				<label>Refresh Interval</label>
+				<description>The frequency at which battery, temperature, and humidity data will refresh</description>
+				<default>300</default>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="temperatureCalibration" type="decimal" min="-1.6" max="1.6" groupName="calibration"
+				unit="Cel">
+				<required>false</required>
```

Comment:
element `required` is deprecated. Use attribute. But in this case as it's the same as the default value you can just remove it. Same for other parameter.

Reply:
done